cpp/log/log_signer.cc

```diff
@@ -136,26 +133,25 @@ LogSigner::SignResult LogSigner::SignTreeHead(SignedTreeHead* sth) const {
 }
 
 // static
-LogSigner::SignResult LogSigner::GetSerializeError(
-    Serializer::SerializeResult result) {
+LogSigner::SignResult LogSigner::GetSerializeError(SerializeResult result) {
   SignResult sign_result;
   switch (result) {
-    case Serializer::INVALID_ENTRY_TYPE:
+    case SerializeResult::INVALID_ENTRY_TYPE:
```

Comment:
Manually mapping enum values like this seems a bit messy. Can defer fixing it to a later PR.